modules/social_features/social_comment/config/install/core.entity_form_display.comment.comment.default.yml

```diff
@@ -10,13 +10,18 @@ bundle: comment
 mode: default
 content:
   field_comment_body:
-    weight: 3
+    weight: 0
     settings:
       rows: 5
       placeholder: 'Add a comment'
     third_party_settings: {  }
     type: string_textarea
     region: content
+  author:
+    weight: 1
+    settings: {  }
+    third_party_settings: {  }
+    region: content
 hidden:
   subject: true
   author: true
```

Comment:
@bramtenhove shouldnt we remove `author` from `hidden` when we add it to the `content`?

Reply:
Nice find! Looks like I put back that part manually 😇